### ISSUE TITLE
best_params_ not supported by RandomizedSearchCV()

### ISSUE THREAD

**Issue by shaunswanson** [databricks/spark-sklearn#105]
The documentation for RandomizedSearchCV implies that a best_params_ property is available after .fit() is called. This does not appear to be the case.

Here is the documentation in question:

https://github.com/databricks/spark-sklearn/blob/master/python/spark_sklearn/random_search.py#L162

Should I attempt to implement a best_params_ property or indicate in the documentation that best_params_ is not supported (as it is a common API in sklearn-related libraries) in a PR?

**Comment by srowen** [comment]
Yes I think this is basically https://github.com/databricks/spark-sklearn/issues/73 which I don't know how to resolve. It's not clear we can set this field. Yeah the docs should be fixed if not.
